src/index.js

```diff
@@ -337,6 +352,16 @@ function getHours(openingHours, closingHours) {
   }
 }
 
+///////////
+// returns a an array of urls as strings if there is a match otherwise null
+// e.g. ['https://google.com']
+///////////
+function extractUrl(item) {
+  // regex source: https://stackoverflow.com/questions/3809401/what-is-a-good-regular-expression-to-match-a-url
+  let url_pattern = /(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})/gi;
```

Comment:
That is a beast of a regex! I'm glad you linked to the stack overflow on it. If we had a unit testing setup, I'd probably ask you to drop a couple examples there but I'm fine going forward with this after a bit of QA.

Reply:
ooo I'll put up a separate PR to unit test this! Great suggestion! hahah yeah it is a scary looking one 😵